src/UIUtils.cc

```diff
@@ -176,6 +176,22 @@ void UIUtils::openUserManualURL()
     openVersionedURL("https://www.openscad.org/documentation.html?version=%1");
 }
 
+bool UIUtils::hasOfflineUserManual(bool openManual = false)
```

Comment:
Can you provide a rationale for these bool returning functions labeled "has" being reused as the routine which opens the manual and cheatsheet?  Generally a function should do only what it says on the label, unless there is a compelling reason otherwise.